test/Core.Test/Services/DeviceServiceTests.cs

```diff
@@ -33,4 +37,61 @@ public async Task DeviceSaveShouldUpdateRevisionDateAndPushRegistration()
         await pushRepo.Received().CreateOrUpdateRegistrationAsync("testtoken", id.ToString(),
             userId.ToString(), "testid", DeviceType.Android);
     }
+
+    /// <summary>
+    /// 
+    /// </summary>
+    [Theory, BitAutoData]
+    public async Task UpdateDevicesTrustAsync_Works(
+        SutProvider<DeviceService> sutProvider,
+        Guid currentUserId,
+        Device deviceOne,
+        Device deviceTwo,
```

Comment:
How are you guaranteeing that `deviceOne` and `deviceTwo` are trusted for the sake of this test as that is a pre req for the logic you are testing?

Reply:
I was leaning on `BitAutoData` filling in data for any string property but I like the idea of being explicit about it so I added a method for doing that.